### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Localization/Basic.lean [hunk 1]
```diff
@@ -811,17 +811,17 @@ theorem isLocalization_iff_of_algEquiv [Algebra R P] (h : S ≃ₐ[R] P) :
 #align is_localization.is_localization_iff_of_alg_equiv IsLocalization.isLocalization_iff_of_algEquiv
 
 theorem isLocalization_iff_of_ringEquiv (h : S ≃+* P) :
-    IsLocalization M S ↔ have := (h.toRingHom.comp <| algebraMap R S).toAlgebra;
-      IsLocalization M P :=
+    IsLocalization M S ↔
+      have := (h.toRingHom.comp <| algebraMap R S).toAlgebra; IsLocalization M P :=
```
Comment (erdOne) [inline Mathlib/RingTheory/Localization/Basic.lean:815]: This should be haveI.
Same for the one below.

File: Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean [hunk 2]
```diff
@@ -101,11 +101,11 @@ theorem TopologicalSpace.IsTopologicalBasis.borel_eq_generateFrom [TopologicalSp
   borel_eq_generateFrom_of_subbasis hs.eq_generateFrom
 #align topological_space.is_topological_basis.borel_eq_generate_from TopologicalSpace.IsTopologicalBasis.borel_eq_generateFrom
 
-theorem isPiSystem_isOpen [TopologicalSpace α] : IsPiSystem (IsOpen : Set α → Prop) :=
+theorem isPiSystem_isOpen [TopologicalSpace α] : IsPiSystem ({ s : Set α | IsOpen s }) :=
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean:104]:
```suggestion
theorem isPiSystem_isOpen [TopologicalSpace α] : IsPiSystem ({s : Set α | IsOpen s}) :=
```
Isn't the convention that, when defining sets, we don't put spaces around the braces? Same thing just below.

**Review by sgouezel — COMMENTED** [review]
bors d+
Thanks!

File: Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean [hunk 3]
```diff
@@ -101,11 +101,11 @@ theorem TopologicalSpace.IsTopologicalBasis.borel_eq_generateFrom [TopologicalSp
...
   fun _s hs _t ht _ => IsOpen.inter hs ht
 #align is_pi_system_is_open isPiSystem_isOpen
 
-lemma isPiSystem_isClosed [TopologicalSpace α] : IsPiSystem (IsClosed : Set α → Prop) :=
+lemma isPiSystem_isClosed [TopologicalSpace α] : IsPiSystem ({ s : Set α | IsClosed s }) :=
```
Comment (Ruben-VandeVelde) [inline Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean:108]:
```suggestion
lemma isPiSystem_isClosed [TopologicalSpace α] : IsPiSystem ({s : Set α | IsClosed s}) :=
```

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**